developer-extension/src/panel/components/tabs/eventsTab/facetList.tsx

```diff
@@ -82,19 +122,41 @@ function FacetValue({
   facetRegistry: FacetRegistry
   excludedFacetValues: ExcludedFacetValues
   onExcludedFacetValuesChange: (newExcludedFacetValues: ExcludedFacetValues) => void
+  addFacetValues: (facet: Facet, facetValue: FacetValue) => void
+  allFacetValues: ExcludedFacetValues
+  parentList: string[]
+  onlyFacet: FacetValue | null
+  setOnlyFacet: (facet: FacetValue | null) => void
 }) {
+  addFacetValues(facet, facetValue)
+
   const isTopLevel = depth === 0
   const isSelected = !excludedFacetValues[facet.path] || !excludedFacetValues[facet.path].includes(facetValue)
+  const isOnly = onlyFacet === facetValue
   const value = (
     <Flex justify="space-between" mt={isTopLevel ? 'xs' : SPACE_BETWEEN_CHECKBOX}>
       <Checkbox
         label={facet.values?.[facetValue]?.label ?? facetValue}
-        checked={isSelected}
+        checked={isSelected && !onlyFacet}
         onChange={() => {
           onExcludedFacetValuesChange(toggleExcludedFacetValue(facet, excludedFacetValues, facetValue))
         }}
       />
-      <Text>{facetValueCount}</Text>
+      <Flex justify="space-between" gap="md">
+        <Text>{facetValueCount}</Text>
+        <Button
+          variant={isOnly ? 'filled' : 'light'}
+          size="compact-xs"
+          w="40px"
+          disabled={!!(onlyFacet && onlyFacet !== facetValue)}
```

Comment:
With previous suggestion we can let all only/button enabled.

```suggestion
        disabled={!!onlyFacet}
```